packages/core/test/unit/html/NodeProcessor.test.js

```diff
@@ -20,6 +20,8 @@ const processAndVerifyTemplate = (template, expectedTemplate, postProcess = fals
     const nodeProcessor = getNewDefaultNodeProcessor();
 
     if (postProcess) {
+      // need to process node first (convert slot to v-slot) before doing post-processing
+      dom.forEach(node => nodeProcessor.processNode(node, new Context(path.resolve(''))));
```

Comment:
Post-processing involves adding of panel id so we have to process the node first before doing post-processing.